pkg/networkpath/traceroute/winconn/winconn_windows_test.go

```diff
@@ -0,0 +1,279 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+//go:build test
+
+package winconn
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"net"
+	"strings"
+	"testing"
+	"time"
+
+	"github.com/DataDog/datadog-agent/pkg/networkpath/traceroute/common"
+	"github.com/DataDog/datadog-agent/pkg/networkpath/traceroute/testutils"
+	"github.com/google/gopacket/layers"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"golang.org/x/net/ipv4"
+	"golang.org/x/sys/windows"
+)
+
+var (
+	srcIP = net.ParseIP("1.2.3.4")
+	dstIP = net.ParseIP("5.6.7.8")
+)
+
+func Test_listenPackets(t *testing.T) {
```

Comment:
nit: haven't seen this pattern before. Usually this would be `TestListenPackets`.

Reply:
Yeah I'm planning to go through and bolster the tests over the next week. I'll be renaming these tests as part of that :)